plugin/imanager.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package plugin
+
+type IManager interface {
```

Comment:
This should have a godoc comment

Reply:
Added the GoDoc similar to `veraison/service/plugin`